lib/WTSI/NPG/iRODS.pm

```diff
@@ -1442,7 +1442,7 @@ sub _replace_object {
       $self->logconfess("Invalid checksum action '$checksum_action'");
   }
   else {
-    $checksum_action = $CALC_CHECKSUM;
+    $checksum_action = $SKIP_CHECKSUM;
```

Comment:
This changes the default action, which previously was to calculate the ckecksum and is still documented as such. Will the change of default affect existing code?

Reply:
This caught my eye too and was the change I brought up in Wednesday's meeting: I think @kjsanger described it as a real bug which had been hidden in a SKIP in the tests until now.

Reply:
We now have time to test and fix anything downstream if needed. The POD for this function has to reflect the default change.